src/scripts/browserDeviceProfile.js

```diff
@@ -794,20 +736,13 @@ define(['browser'], function (browser) {
             ]
         });
 
-        if (!browser.edgeUwp && !browser.tizen && !browser.orsay && !browser.web0s) {
-            //profile.CodecProfiles[profile.CodecProfiles.length - 1].Conditions.push({
-            //    Condition: 'NotEquals',
-            //    Property: 'IsAVC',
-            //    Value: 'false',
-            //    IsRequired: false
-            //});
-
-            //profile.CodecProfiles[profile.CodecProfiles.length - 1].Conditions.push({
-            //    Condition: 'NotEquals',
-            //    Property: 'IsInterlaced',
-            //    Value: 'true',
-            //    IsRequired: false
-            //});
+        if (!browser.edgeUwp && !browser.tizen && !browser.web0s) {
+            profile.CodecProfiles[profile.CodecProfiles.length - 1].Conditions.push({
+                Condition: 'NotEquals',
+                Property: 'IsInterlaced',
+                Value: 'true',
+                IsRequired: false
+            });
```

Comment:
This likely needs testing, mainly to ensure that it actually does something for interlaced video.